charts/workflows/charts/move2kube/templates/02-configmap_m2k-props.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+data:
+  application.properties: |2-
+
+    # Configuration for the incoming cloud events received by the serverless workflows.
+    mp.messaging.incoming.kogito_incoming_stream.connector=quarkus-http
+    mp.messaging.incoming.kogito_incoming_stream.path=/
+    mp.messaging.incoming.kogito_incoming_stream.method=POST
+
+
+    move2kube_url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
+    quarkus.rest-client.move2kube_yaml.url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
+    broker_url=${BROKER_URL:http://broker-ingress.knative-eventing.svc.cluster.local/m2k/default}
+    quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
```

Comment:
default should be the default name of janus service which according to the orchestrator-helm should be `orchestrator-backstage.default.svc.cluster.local`   (for the default namespace of course)